### PR TITLE
keymaps for planck and crkbd

### PR DESCRIPTION
**Pull request by dsanchezseco**
## Description

Updates keymaps

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/crkbd/keymaps/dsanchezseco/keymap.c [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+  _RAISE,
+  _ADJUST
+};
+
+enum custom_keycodes {
+  DVORAK = SAFE_RANGE,
+  LOWER = MO(_LOWER),
+  RAISE = MO(_RAISE),
+  ESC_CTL = LCTL_T(KC_ESC),
+};
```
Comment (drashna) [inline keyboards/crkbd/keymaps/dsanchezseco/keymap.c]: This entire block/enum could be removed.   If you only have DVORAK, you don't need the keycode for it.  

For the rest:
```c
#define LOWER MO(_LOWER)
#define RAISE MO(_RAISE)
#define ESC_CTL LCTL_T(KC_ESC)
```

File: keyboards/crkbd/keymaps/dsanchezseco/keymap.c [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+uint32_t layer_state_set_user(uint32_t state) {
+  return update_tri_layer_state(state, _LOWER, _RAISE, _ADJUST);
+}
+
+#ifdef OLED_DRIVER_ENABLE
+oled_rotation_t oled_init_user(oled_rotation_t rotation) {
+  if (!isLeftHand)
+    return OLED_ROTATION_180;  // flips the display 270 to see it from my side
```
Comment (drashna) [inline keyboards/crkbd/keymaps/dsanchezseco/keymap.c]: Don't want to set to 270?

File: keyboards/planck/keymaps/dsanchezseco/keymap.c [hunk 3]
```diff
@@ -12,83 +12,40 @@ enum planck_layers {
 
 enum planck_keycodes {
   DVORAK = SAFE_RANGE,
+  ESC_CTL = LCTL_T(KC_ESC),
 };
```
Comment (drashna) [inline keyboards/planck/keymaps/dsanchezseco/keymap.c]: Same as above. You don't need this block.

File: keyboards/crkbd/keymaps/dsanchezseco/keymap.c (hunk 2 above)
Comment (dsanchezseco) [reply keyboards/crkbd/keymaps/dsanchezseco/keymap.c]: My bad, outdated comment

File: keyboards/crkbd/keymaps/dsanchezseco/keymap.c (hunk 1 above)
Comment (dsanchezseco) [reply keyboards/crkbd/keymaps/dsanchezseco/keymap.c]: Thanks!

**Pull request merged**